// stripe/cjs/resources/Issuing/CreditUnderwritingRecords.js
"use strict";
// File generated from our OpenAPI spec
Object.defineProperty(exports, "__esModule", { value: true });
exports.CreditUnderwritingRecords = void 0;
const StripeResource_js_1 = require("../../StripeResource.js");
const stripeMethod = StripeResource_js_1.StripeResource.method;
exports.CreditUnderwritingRecords = StripeResource_js_1.StripeResource.extend({
    retrieve: stripeMethod({
        method: 'GET',
        fullPath: '/v1/issuing/credit_underwriting_records/{credit_underwriting_record}',
    }),
    list: stripeMethod({
        method: 'GET',
        fullPath: '/v1/issuing/credit_underwriting_records',
        methodType: 'list',
    }),
    correct: stripeMethod({
        method: 'POST',
        fullPath: '/v1/issuing/credit_underwriting_records/{credit_underwriting_record}/correct',
    }),
    createFromApplication: stripeMethod({
        method: 'POST',
        fullPath: '/v1/issuing/credit_underwriting_records/create_from_application',
    }),
    createFromProactiveReview: stripeMethod({
        method: 'POST',
        fullPath: '/v1/issuing/credit_underwriting_records/create_from_proactive_review',
    }),
    reportDecision: stripeMethod({
        method: 'POST',
        fullPath: '/v1/issuing/credit_underwriting_records/{credit_underwriting_record}/report_decision',
    }),
});
